Examples/zib-Product-04.xml

```diff
@@ -0,0 +1,20 @@
+<Medication xmlns="http://hl7.org/fhir">
+    <id value="zib-product-04/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Product"/>
+    </meta>
+    <code>
+        <coding>
```

Comment:
Ik twijfel of we dit zouden moeten doen. Kunnen we bij Marijke niet wat realistische codes vragen?

Reply:
Dat kan ik inderdaad even aan haar vragen.